### DISCUSSION TITLE
Removing/Uninstalling netbox_topology_views

### DISCUSSION THREAD

**Discussion by mhabiballa** [mattieserver/netbox-topology-views#469] [Q&A] [answered]
Hi,
How to,please?

**Comment by BadiaJunkie** [comment]
Try something like this:

sudo -i
cd projects (my setup)
cd netbox-topology-views
ls
source /opt/netbox/venv/bin/activate
pip uninstall netbox-topology-views

Don't forget to edit the local_requirements.txt and configuration.py

**Comment by cs-1** [comment]
Sorry for bringing this old topic up: are there any tables etc. that would need to be removed from the database to uninstall the plugin trace-free?

**Comment by dreng** [comment, marked as answer]
Hi,

just remove all tables that start with "netbox_topology_views".
